modules/apps/journal/journal-web/src/main/resources/META-INF/resources/article/basic_info.jsp

```diff
@@ -93,34 +93,43 @@ DDMStructure ddmStructure = journalEditArticleDisplayContext.getDDMStructure();
 
 <c:choose>
 	<c:when test="<%= !journalWebConfiguration.journalArticleForceAutogenerateId() && (journalEditArticleDisplayContext.getClassNameId() == JournalArticleConstants.CLASS_NAME_ID_DEFAULT) %>">
+
+		<%
+		boolean existingArticle = (article != null) && !article.isNew();
```

Comment:
Better use `isNew` / `isNewArticle` instead of `existingArticle ` is more coomnd in our code base.